gemfiles/rails42.gemfile

```diff
@@ -28,7 +28,7 @@ end
 
 group :test do
   gem "ammeter"
-  gem "capybara", "3.30.0"
```

Comment:
Is the Capybara update relevant?

Reply:
No. I'll remove it before I merge it. I'm most interested here on whether or not it makes a build difference / actually works